modules/programs/inori.nix

```diff
@@ -0,0 +1,47 @@
+{ lib, config, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.programs.inori;
+
+  tomlFormat = pkgs.formats.toml { };
+in {
+  meta.maintainers = [ hm.maintainers.lunahd ];
```

Comment:
```suggestion
  meta.maintainers = with hm.maintainers; [ lunahd stephen-huan ];
```

Reply:
then just

```nix
meta.maintainers = [ hm.maintainers.lunahd lib.maintainers.stephen-huan ];
```

should be fine; the `with lib.maintainers;` is just a common nixpkgs-ism.